tests/unit/rl_worker/test_worker_util.py

```diff
@@ -0,0 +1,172 @@
+        expected = "{0}{1}".format(EVALAI_API_SERVER, self.make_request_url())
+        self.assertEqual(returned_url, expected)
+
+    def test_get_message_from_sqs_queue(self):
+        url = "{}{}".format(EVALAI_API_SERVER, self.get_message_from_sqs_queue_url(EVALAI_QUEUE_NAME))
+        response = self.interface.get_message_from_sqs_queue()
+
+        self.mock_make_request.assert_called_with(url, "GET")
+        self.assertEqual(response, self.success_response)
+
+    def test_get_submission_by_pk(self):
+        url = "{}{}".format(EVALAI_API_SERVER, self.get_submission_by_pk_url(self.submission_pk))
+        response = self.interface.get_submission_by_pk(self.submission_pk)
+
+        self.mock_make_request.assert_called_with(url, "GET")
+        self.assertEqual(response, self.success_response)
+
+    def test_get_challenge_phases_by_challenge_pk(self):
+        url = "{}{}".format(EVALAI_API_SERVER, self.get_challenge_phases_by_challenge_pk_url(self.challenge_pk))
+        response = self.interface.get_challenge_phases_by_challenge_pk(self.challenge_pk)
+
+        self.mock_make_request.assert_called_with(url, "GET")
+        self.assertEqual(response, self.success_response)
+
+    def test_get_challenge_by_queue_name(self):
+        url = "{}{}".format(EVALAI_API_SERVER, self.get_challenge_by_queue_name_url(EVALAI_QUEUE_NAME))
+        response = self.interface.get_challenge_by_queue_name()
+
+        self.mock_make_request.assert_called_with(url, "GET")
+        self.assertEqual(response, self.success_response)
+
+    def test_get_challenge_phase_by_pk(self):
+        url = "{}{}".format(EVALAI_API_SERVER, self.get_challenge_phase_by_pk_url(self.challenge_pk, self.challenge_phase_pk))
+        response = self.interface.get_challenge_phase_by_pk(self.challenge_pk, self.challenge_phase_pk)
+
+        self.mock_make_request.assert_called_with(url, "GET")
+        self.assertEqual(response, self.success_response)
+
+    def test_update_submission_data(self):
+        url = "{}{}".format(EVALAI_API_SERVER, self.update_submission_data_url(self.challenge_pk))
+        data = {"submission_pk": self.submission_pk, "test_field": "new_value"}
+        response = self.interface.update_submission_data(data, self.challenge_pk, self.submission_pk)
+
+        self.mock_make_request.assert_called_with(url, "PUT", data=data)
+        self.assertEqual(response, self.success_response)
+
+    def test_update_submission_data_partially(self):
+        url = "{}{}".format(EVALAI_API_SERVER, self.update_submission_status_url(self.challenge_pk))
+        data = {"test_field": "new_value"}
+        response = self.interface.update_submission_status(data, self.challenge_pk)
+
+        self.mock_make_request.assert_called_with(url, "PATCH", data=data)
+        self.assertEqual(response, self.success_response)
+
+    def test_delete_message_from_sqs_queue(self):
+        test_receipt_handle = "MbZj6wDWli+JvwwJaBV+3dcjk2YW2vA3+STFFljTM8tJJg6HRG6PYSasuWXPJB+Cw"
```

Comment:
Is this random string? If yes, how about using replacing hard coded string with a randomize function for alphanumeric and symbols?

Reply:
Nice idea! I will create a helper a helper for that.

Reply:
Done! 👍